spec/rubocop/cop/rails/where_not_with_multiple_arguments_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::WhereNotWithMultipleArguments, :config do
+  it 'does not register an offense for where.not with one argument' do
+    expect_no_offenses(<<~RUBY)
+      User.where.not(trashed: true)
+    RUBY
+  end
+
+  it 'registers an offense for where.not with multiple arguments' do
+    expect_offense(<<~RUBY)
+      User.where.not(trashed: true, role: 'admin')
+           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use a SQL statement instead of where.not with multiple arguments.
```

Comment:
As https://github.com/rails/rails/pull/36029 shows, can you update the message as follows?

```suggestion
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `where.not(trashed: true).where.not(role: 'admin')` instead.
```

Reply:
The thing is: If this line was written with the behavior of Rails 6.1+ in mind and the developer wanted all users that aren't trashed admins, then `where.not(trashed: true).where.not(role: 'admin')` is wrong.
In this case, I would still prefer `where.not('trashed = ? AND role = ?', true, :admin)` for clarity, but this is the reason why I didn't add any corrections